Realm.Shared/native/NativeTable.cs

```diff
@@ -28,104 +28,103 @@ namespace Realms
     [SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1121:UseBuiltInTypeAlias")]
     internal static class NativeTable
     {
-        public static IntPtr AddEmptyRow(TableHandle tableHandle)
+        public static IntPtr AddEmptyRow(TableHandle tableHandle, SharedRealmHandle sharedRealm)
```

Comment:
Maybe we need to make a new `CreateEmptyObject(TableHandle)` method on the `SharedRealmHandle` class instead.